src/test/java/wbooks/login/login.feature

```diff
@@ -0,0 +1,37 @@
+Feature: enter the credentials to get a valid user token
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+  @login
+  Scenario: login with valid credentials and get 200 with user token
+    Given path 'users/sessions'
+      And request
+      """
+      {
+        "session": {
+            "email": "wolox@test.com.ar",
+            "password": "12345678"
+        }
+      }
+      """
+    When method post
+    Then status 200
+      And match response == { access_token: '#string', renew_id: '#string' }
+
+    * def access_token = response.access_token
+    * def renew_id = response.renew_id
```

Comment:
You can put this as a step on scenario. That is, after the match as AND access_token....
So that they do not remain as defined variables in the middle of the scenarios.

Reply:
Mmm I think I dont quite get this one.
Both already are a step in the @login scenario ("login with valid credentials and get 200 with user token") and only for that one, Im just using the wildcard `*` as its not really important for the assertion (`Then`) and its just there to easier reuse in future scenarios.

Reply:
I mean putting those two variables after the match, that is:
And match ...
And access_token ...
And renew_id ...
They are part of the same scenario and not as loose variables. So you can also use them in other features.

Reply:
But they are still part of the same scenario this way and they can be used in other features using a call this way.
I just chose to separate them because, even they are part of the scenario, they aren't part of the assert, they are there just to be used in other features.

I don't know, I could change them that way but I think using `And` for those after the `Then` sentence might confuse the reader to think they are still doing asserts when in reality the assert is already over.

Reply:
Well!, you can leave it like that, I say it more than anything in order and as "best practices" although there is not something standardized as best practice in KarateDSL, but just as it works it works perfectly!.
Done ;-)